docs/quickstart/live-demo.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Live demo
+enableToc: false
+---
+
+# Before you start
+
+You will need an OpenAPI specification. For testing purposes, you can use one of the following sample OpenAPI specifications: 
+- OpenAPI 3.0
+    - [Rebilly Users OpenAPI Specification](https://raw.githubusercontent.com/Rebilly/api-definitions/master/openapi/users.yaml)
+    - [Swagger Petstore Sample OpenAPI Specification](https://petstore3.swagger.io/api/v3/openapi.json)
+- OpenAPI 2.0
+    - [Thingful OpenAPI Specification](https://raw.githubusercontent.com/thingful/openapi-spec/master/spec/swagger.yaml)
+    - [Fitbit Plus OpenAPI Specification](https://raw.githubusercontent.com/TwineHealth/TwineDeveloperDocs/master/spec/swagger.yaml)
+
+For more information on OpenAPI specifications see [Learning OpenAPI 3](https://redoc.ly/docs/resources/learning-openapi/).
+
+## Live demo online
+
+If you want to see how ReDoc will render your OpenAPI specification, you can try it out online at https://redocly.github.io/redoc/.
+
+A version of the Swagger Petstore API is already displayed.  Try with your own OpenAPI specification by
+entering the URL for your specification and clicking the **TRY IT** button.
```

Comment:
```suggestion
enter the URL for your specification and select the **TRY IT** button.
```

Reply:
fixed

Reply:
@HCloward - For reference, since we use the Microsoft Style guide, we use `select` instead of `click`